pulsar-broker/src/test/java/org/apache/pulsar/broker/s3offload/S3ManagedLedgerOffloaderTest.java

```diff
@@ -18,30 +18,40 @@
  */
 package org.apache.pulsar.broker.s3offload;
 
+import static org.mockito.Matchers.any;
+import static org.testng.Assert.assertEquals;
```

Comment:
why import static when Assert is already pulled in? I've seen other people seem to prefer the static as well. I'm just wondering what the reason behind it is.

Reply:
Intellij Idea does this automatically. Seems it will save some typing of `Assert` before `assertEquals`